Guides/Getting Started.md

```diff
@@ -2,15 +2,15 @@
 
 ## Preface
 
-If you are using Orion for tweak development, it is recommended that you use it with Theos. This guide assumes that you have Theos installed; if you haven't done that yet, please follow the [installation instructions](https://github.com/theos/theos/wiki/Installation) for Theos and then return to this guide.
+If you are using Orion for tweak development, it is recommended that you use it with Theos. This guide assumes that you have Theos installed; if you haven't done that yet, please follow the [installation instructions](https://theos.dev/docs/installation) for Theos and then return to this guide.
 
 If you wish to use Orion without Theos, please refer to the "[Using Orion Without Theos](using-orion-without-theos.html)" guide.
 
 This guide will show you how to make a simple Orion tweak which… spices up text labels a little. We will target the VLC Media Player iOS app since it is [open source](https://github.com/videolan/vlc-ios) and supports a large range of iOS versions, however most other apps should work too.
 
 To follow along, you will require the following things:
 
-- [Theos](https://github.com/theos/theos/wiki/Installation) on a machine running macOS, Windows 10 with WSL, or Linux.
+- [Theos](https://github.com/theos/theos/wiki/Installation) on a machine running macOS, Windows 10 with WSL, or Linux. If you're on a Linux environment, please make sure to install the Swift package from [here](https://www.swift.org/download/).
```

Comment:
Let's update the wiki link to theos.dev here too. Also, I'd modify the wording a little:
```suggestion
- [Theos](https://theos.dev/docs/installation) on a machine running macOS, Windows 10 with WSL, or Linux. If you're in a Linux environment, you must install a [host Swift toolchain](https://www.swift.org/download/) in addition to a Swift-compatible iOS toolchain.
```

Reply:
Yep, that sounds better indeed. Committed the requested changes.